save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ProjectView.kt

```diff
@@ -628,6 +628,15 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
                     testingType = state.testingType
                     isSubmitButtonPressed = state.isSubmitButtonPressed
                     gitDto = gitDto
+                    projectName = props.name
+                    organizationName = props.owner
+                    onContestEnrollerResponse = {
+                        setState {
+                            isErrorOpen = true
+                            errorMessage = it
+                            errorLabel = "Contest enrollment"
```

Comment:
Why are you opening an error modal?

Reply:
Because this is a modal with the only button
#803 :)

Reply:
OK